src/support/action/acceptCookieBanner.js

```diff
@@ -0,0 +1,13 @@
+// To dismiss cookie banner, as it blocks clicks
+
+module.exports = (text) => {
```

Comment:
Can you add a corresponding step definition inside https://github.com/nice-digital/wdio-cucumber-steps/blob/master/src/when/definitions.js#L25 pls? And then run the readme command to update the readme with the new command?

Reply:
Thanks, totally missed this. So this is added - I don't know where the 'readme command' is. This might need another update.